mender/values.yaml

```diff
@@ -239,14 +239,14 @@ api_gateway:
   env:
     SSL: false
   minio:
-    enabled: true
+    enabled: false
     podAnnotations: {}
     url: "http://minio:9000"
     customRule: null
   storage_proxy:
-    enabled: false
-    url: ""
-    customRule: "HostRegexp(`{domain:^artifacts.*$}`)"
+    enabled: true
+    url: "http://seaweedfs-s3:8333"
+    customRule: "PathRegexp(`^/mender-artifact-storage`)"
```

Comment:
I chose this strategy because I found very annoying the way we where exposing the services with MinIO/SeaweedFS:

You create MinIO/SeaweedFS, you expose the service somewhat both internally and externally (artifacts.example.com). If you're using an EC2 and KinD you have to tweak the /etc/hosts to point artifacts.example.com to your Docker interface.
So I simply chose to use the storage_proxy, and just the Mender Ingress. The storage is reachable through `https://mender.example.com/my-storage-bucket` and that's it.

Reply:
Agreed, this is cleaner. I believe the reason it was done this way was that deployments service used to call ListAllMyBuckets on startup (which would not provide the bucket name in the path). It's been several releases since it was replaced with HeadBucket instead.